core/trino-parser/src/main/java/io/trino/sql/ReservedIdentifiers.java

```diff
@@ -121,24 +142,12 @@ private static void validateDocs(Path path)
 
     public static Set<String> reservedIdentifiers()
     {
-        return sqlKeywords().stream()
-                .filter(ReservedIdentifiers::reserved)
-                .sorted()
-                .collect(toImmutableSet());
+        return RESERVED_IDENTIFIERS;
```

Comment:
Remove the methods `reservedIdentifiers()` and `sqlKeywords()`, and use the constants instead. `SQL_KEYWORDS` needs to be made public.